app/src/pages/authoring/index.tsx

```diff
@@ -16,28 +16,54 @@ const useStyles = createStyles(() => ({
 
 export default function AuthoringPage() {
   const [resourceType, setResourceType] = useState<ArtifactResourceType>('Measure');
+  const [selectedArtifact, setSelectedArtifact] = useState<string | null>(null);
+
+  const { data: artifacts } = trpc.service.getArtifactsByResource.useQuery({ resourceType });
 
   const ctx = trpc.useContext();
   const { classes } = useStyles();
 
+  const successNotification = (data: { draftId: string }, cloned: boolean) => {
+    const title = cloned ? `${resourceType} Cloned!` : `${resourceType} Created!`;
+    const message = cloned ? `${resourceType} successfully cloned` : `${resourceType} successfully created`;
+    notifications.show({
+      title: title,
+      message: message,
+      icon: <CircleCheck />,
+      color: 'green'
+    });
+    router.push(`authoring/${resourceType}/${data.draftId}`);
+    ctx.draft.getDraftCounts.invalidate();
+  };
+
+  const errorNotification = (errorMessage: string, cloned: boolean) => {
+    const title = cloned ? `${resourceType} Clone Failed!` : `${resourceType} Creation Failed!`;
+    const message = cloned
+      ? `Attempt to clone ${resourceType} failed with message: ${errorMessage}`
+      : `Attempt to create ${resourceType} failed with message: ${errorMessage}`;
+    notifications.show({
+      title: title,
+      message: message,
+      icon: <AlertCircle />,
+      color: 'red'
+    });
+  };
+
```

Comment:
You also have a success/error notification pop up in `[app/src/pages/[resourceType]/[id].tsx]` on lines 39-57 and I was wondering if it would be helpful to use/modify these two methods: errorNotification() and successNotification() in that file as well to reduce code?

Reply:
Good suggestion! I think I will pull them out into a `util/draftArtifactNotifications` file if that makes sense.

Reply:
Decided to keep this is way it is based on our conversation in dev hour.